app/models/doc_type.rb

```diff
@@ -9,7 +9,7 @@ def initialize(id, sequence, name)
 
   DOCTYPES = [
     DocType.new(1,  500,  'Representation order'),
-    DocType.new(2,  200,  'LAC1 - memo of conviction'),
+    # DocType.new(2,  200,  'LAC1 - memo of conviction'),
```

Comment:
Can this be deleted too?

Reply:
can do, will add it to the one commit i have for removing the LAC1 evidence checklist item, in case we need to revert or cherry pick later.